src/components/PageContainer/PageContainer.tsx

```diff
@@ -17,7 +18,8 @@ export const PageContainer = ({ children }: PageContainerProps) => {
 
   const redirectToProfile = () => {
     const cleanHostname = window.location.hostname.replace('am.ui.', '');
-    window.location.href = 'https://' + cleanHostname + '/' + ApiDelegationPath.Profile;
+    window.location.href =
+      'https://' + cleanHostname + '/' + GeneralPath.Profile + '?R=' + getCookie('AltinnPartyId');
```

Comment:
I'm assuming this is to keep the user as the same party (org) when sending them back to the profile?
(Hard to test this locally so I'm just asking 😅)

Reply:
Yeah it is because there's a bug in Altinn2